packages/ui/src/components/va-virtual-scroller/VaVirtualScroller.stories.ts

```diff
@@ -1,12 +1,285 @@
 import { defineComponent } from 'vue'
-import VaVirtualScroller from './VaVirtualScroller.demo.vue'
+import { within } from '@storybook/testing-library'
+import { expect } from '@storybook/jest'
+
+import { VaVirtualScroller } from './'
+import { VaBadge } from '../va-badge'
+import { VaButton } from '../va-button'
 
 export default {
   title: 'VaVirtualScroller',
   component: VaVirtualScroller,
+  tags: ['autodocs'],
+}
+
+const hugeArrayBase = new Array(1000)
+const hugeArray = hugeArrayBase.fill(null).map((_, index) => index)
+const hugeObjectsArray = hugeArrayBase.fill(null).map((el, index) => ({ value: index }))
+
+export const Default = () => ({
+  components: { VaVirtualScroller },
+  data: () => ({ hugeArray }),
+  template: `
+    <va-virtual-scroller
+      :items="hugeArray"
+      :wrapper-size="200"
+      v-slot="{item}"
+    >
+      <div>{{ item }}</div>
+    </va-virtual-scroller>
+  `,
+})
+
+Default.play = async ({ step }) => {
+  const scroller = document.querySelector('.va-virtual-scroller') as HTMLElement
```

Comment:
I suggest we use `canvasElement.querySelector` instead of `document`. It's more local.